tests/js/server/aql/aql-functions-date.js

```diff
@@ -2457,6 +2457,22 @@ function ahuacatlDateFunctionsTestSuite () {
 	  assertEqual([ "2020-02-29T23:00:00.000Z" ], 
 		getQueryResults("RETURN DATE_LOCALTOUTC(DATE_ADD(DATE_UTCTOLOCAL('2020-01-31T23:00:00.000Z', 'Europe/Berlin'), 1, 'months'), 'Europe/Berlin')"));	  
     },
+	
+	// //////////////////////////////////////////////////////////////////////////////
+    // / @brief test date_timezone
+    // //////////////////////////////////////////////////////////////////////////////
+
+    testDateTimeZone: function () {
+	  getQueryResults("RETURN DATE_TIMEZONE()", {});
+    },
+	
+	// //////////////////////////////////////////////////////////////////////////////
+    // / @brief test date_timezones
+    // //////////////////////////////////////////////////////////////////////////////
+	
+	testDateTimeZones: function () {
+	  getQueryResults("RETURN DATE_TIMEZONES()", {});
```

Comment:
This does not perform any checks. I know that the list of tzs can change over time, but it is at least assumed to be always an array.
In addition, as the timezones are part of the repository now, we can also assume that this function returns timezones that are present in the tzdata in 3rdParty.
At least that is my understanding of how it works: by providing the tzdata in 3rdParty we have control over which tz names will be returned by this function.
If so, IMHO we should add a test here that checks if a few select timezone names are part of the result list.